doc/source.md

```diff
@@ -0,0 +1,221 @@
+# Source
+
+The Kafka Connect TDengine Source connector is used to move creations in a TDengine database to Apache Kafka® topics in
+real-time. Data is loaded by periodically executing query, the connector loads only new records. By default, all super
+tables in a database are copied, each to its own output topic.
+
+## Start
+
+This example assumes you are running Confluent version 7.1.1 locally on the default ports. It also assumes your have
+TDengine installed and running.
+
+### Start TDengine Source Connector
+
+1. Start the Confluent Platform using the Confluent CLI command below.
+
+   ```shell
+   confluent local services start
+   ```
+
+2. create a configuration file for the connector. This configuration is used typically with standalone workers. This
+   file is included with the connector in `config/source-quickstart.properties`, and contains the following settings:
+
+   ```properties
+   name=tdengine-source
+   connector.class=com.taosdata.kafka.connect.source.TDengineSourceConnector
+   tasks.max=1
+   connection.url=jdbc:TAOS://127.0.0.1:6030
+   connection.username=root
+   connection.password=taosdata
+   connection.database=source
+   connection.attempts=3
+   connection.backoff.ms=5000
+   
+   poll.interval.ms=1000
+   topic.prefix=tdengine-
+   fetch.max.rows=100
+   out.format=line
+   key.converter=org.apache.kafka.connect.storage.StringConverter
+   value.converter=org.apache.kafka.connect.storage.StringConverter
+
+   ```
+
+   The `connection.url`,`connection.database` specify the connection URL,database name of the TDengine server. By
```

Comment:
need space between "," and following content.

Reply:
do as you say